### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by doctorkb** [njobrien1006/hass_traeger#68] [General]
Just wanted to post a quick note to say "thank you" for continuing to develop this integration.  Given some of the contraints on the original (sebirdman) one, I had given up for a couple of years until I spotted this maintained fork, in part because it appears to be no longer maintained.

Please keep up the good work!

**Comment by rossc719g** [comment]
A big +1 from me!

I had been using the older integration for a long time, and had noticed it getting more and more flaky, and then in the latest upgrade of HASS, it stopped working completely.  I deleted the old one, switched to this one, and all of my existing dashboards and automations simply worked without any tweaking needed.  Impressive.  :-)

So, thanks so much from me too!

**Comment by ahuffman** [comment]
Thanks for maintaining this.  I just got it setup after trying the original hass_traeger integration and finding that nobody seems to maintain it anymore.

**Comment by jianyu-li** [comment]
I came to write my own thank you post but adding to this one. THANK YOU THANK YOU for continued support. I use this every Thanksgiving to make sure my Turkey is perfect. I have been designated the turkey cooker because it comes out so great every year and it's largely due to this integration and sensors I've built from it. Thank you!

**Reply by lymanepp** [reply]
I agree 💯 and, in fact, I bought my Traeger a year ago solely because of this integration.
